VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimDetailsScreen/ClaimStatus/ClaimStatus.test.tsx

```diff
@@ -51,44 +41,48 @@ context('ClaimStatus', () => {
   })
 
   it('should initialize', async () => {
```

Comment:
All the `async`s in this file can be removed

Reply:
removed